app/Busline.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App;
```

Comment:
It would be nice if you could change `App` namespace to something more project-specific, let's say `OpenTransportManager`. ;)

Reply:
Does that help anything in our case though? As Szymon mentioned on Slack, feature to change app namespace has been removed from artisan in Laravel 6, so that would have to be done manually.

https://github.com/laravel/framework/pull/27575
> Having a customized namespace is more something you'd do for a package and not an application.

Reply:
From my experience it's a nice thing for every project. Imagine having four projects opened in different windows and four identical `App\Exceptions\Handler` classes. Of course, you can add some plugin like [this](https://plugins.jetbrains.com/plugin/8032-project-label), but changing global namespace at this moment shouldn't be hard and time consuming. Anyway, it's your call, of course.

Reply:
It may be done easily via PhpStorm. **Edit->Find->Replace in path**.